src/execution/as_abi.rs

```diff
@@ -193,16 +175,7 @@ pub(crate) fn assembly_script_has_op_key(env: &ASEnv, key: i32) -> ABIResult<i32
         param_size_update(env, &fname, key_bytes.len(), true);
     }
 
-    match env.get_interface().has_op_key(&key_bytes) {
-        Err(err) => abi_bail!(err),
-        Ok(b) => {
-            // https://doc.rust-lang.org/reference/types/boolean.html
-            // 'true' is explicitly defined as: 0x01 while 'false' is: 0x00
-            let b_vec: Vec<u8> = vec![b as u8];
-            let a = pointer_from_bytearray(env, &b_vec)?.offset();
```

Comment:
What was this code for ? it's so strange

Reply:
It's basically converting a bool to a byte array to later maker a pointer with it, but all this can be achieved by simply returning the bool as an `i32`